taqueria-protocol/Config.ts

```diff
@@ -101,6 +102,7 @@ export const internalSchema = commonSchema.extend({
 	environment: environmentMap,
 	accounts: accountsMap,
 	contracts: z.record(Contract.schemas.schema).optional(),
+	metadata: MetadataConfig.schemas.schema.optional(),
```

Comment:
Is this correct for a flat optional object that should be at the root of the config.json?

```
{
  ....,
  metadata: {
    name: 'Name'
    ...,
  },
}
```